
{'date': 'Tue Sep 19 2023 11:51:34.800 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 9
  oceano = Cena(IMAGEM).vai()
  ^
IndentationError: expected an indented block
'''},
{'date': 'Tue Sep 19 2023 11:51:57.755 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 9
  oceano = Cena(IMAGEM).vai()
  ^
IndentationError: expected an indented block
'''},
{'date': 'Tue Sep 19 2023 11:52:09.971 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 9
  oceano = Cena(IMAGEM).vai()
  ^
IndentationError: expected an indented block
'''},
{'date': 'Tue Sep 19 2023 11:53:24.164 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 9
  oceano = Cena(IMAGEM).vai()
  ^
IndentationError: expected an indented block
'''},
{'date': 'Tue Sep 19 2023 11:53:40.32 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 9
  oceano = Cena(IMAGEM).vai()
  ^
IndentationError: expected an indented block
'''},
{'date': 'Tue Sep 19 2023 11:54:13.887 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 9
  oceano = Cena(IMAGEM).vai()
  ^
IndentationError: expected an indented block
'''},
{'date': 'Tue Sep 19 2023 11:55:35.875 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 9
  oceano = Cena(IMAGEM).vai()
  ^
IndentationError: expected an indented block
'''},
{'date': 'Tue Sep 19 2023 12:01:26.22 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 10
  oceano = Cena(IMAGEM).vai()
  ^
IndentationError: expected an indented block
'''},
{'date': 'Tue Sep 19 2023 12:22:15.893 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 4
    STYLE["width"] = 800
NameError: name 'STYLE' is not defined
'''},
{'date': 'Tue Sep 19 2023 12:38:32.625 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 20
  def move(self,ev=nome):
  ^
IndentationError: unexpected indent
'''},
{'date': 'Tue Sep 19 2023 12:39:06.810 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 16
    class Peao:
  module <module> line 20
    def move(self,ev=nome):
NameError: name 'nome' is not defined
'''},
{'date': 'Tue Sep 26 2023 11:38:05.737 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 29
    IlhaProibida()
  module <module> line 13
    self.terreno = Terreno(PORTAO_BRONZE, 10, 50, 100, 100, "Portao de Bronze",cena=oceano)
TypeError: __init__() takes 5 positional argument but more were given
'''},
{'date': 'Tue Sep 26 2023 11:38:16.55 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 29
    IlhaProibida()
  module <module> line 13
    self.terreno = Terreno(PORTAO_BRONZE, 10, 50, 100, 100, "Portao de Bronze",cena=oceano)
TypeError: __init__() takes 5 positional argument but more were given
'''},
{'date': 'Tue Sep 26 2023 11:38:33.133 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 29
    IlhaProibida()
  module <module> line 13
    self.terreno = Terreno(PORTAO_BRONZE, 10, 50, 100, 100, "Portao de Bronze",cena=oceano)
TypeError: __init__() takes 5 positional argument but more were given
'''},
{'date': 'Tue Sep 26 2023 11:41:10.343 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 29
    IlhaProibida()
  module <module> line 13
    self.terreno = Terreno(PORTAO_BRONZE, 10, 50, "Portao de Bronze",cena=oceano)
TypeError: __init__() got an unexpected keyword argument 'cena'
'''},
{'date': 'Tue Sep 26 2023 11:43:08.977 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 29
    IlhaProibida()
  module <module> line 13
    self.terreno = Terreno(PORTAO_BRONZE, 10, 50, "Portao de Bronze",cena=oceano)
TypeError: __init__() got an unexpected keyword argument 'cena'
'''},
{'date': 'Tue Sep 26 2023 11:45:03.728 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 29
    IlhaProibida()
  module <module> line 13
    self.terreno1 = Terreno(PORTAO_BRONZE, 10, 50, "Portao de Bronze",cena=oceano)
TypeError: __init__() got an unexpected keyword argument 'cena'
'''},
{'date': 'Tue Sep 26 2023 11:53:23.144 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 29
    IlhaProibida()
  module <module> line 13
    self.terreno1 = Terreno(PORTAO_BRONZE, posx=10, posy=50, titulo="Portao de Bronze", cena=oceano)
TypeError: __init__() got an unexpected keyword argument 'cena'
'''},
{'date': 'Tue Sep 26 2023 11:55:03.450 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 29
    IlhaProibida()
  module <module> line 13
    self.terreno1 = Terreno(PORTAO_BRONZE, posx=10, posy=50, cena=oceano)
TypeError: __init__() got an unexpected keyword argument 'cena'
'''},
{'date': 'Tue Sep 26 2023 12:08:57.738 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 16
  cena=oceano)
        ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 26 2023 12:39:44.882 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 17
  info_terrenos= [PORTAO_BRONZE, PALACIO_CORAL, PORTAO_BRONZE, PALACIO_CORAL]
  ^
IndentationError: unexpected indent
'''},
{'date': 'Tue Sep 26 2023 12:39:57.447 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 42
  def mover(self, x)
                     ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Oct 10 2023 11:38:22.324 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 48
  """ Move o peão para a direita
  ^
IndentationError: expected an indented block
'''},
{'date': 'Tue Oct 10 2023 12:19:36.35 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Executado no modulo julia
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 82
    IlhaProibida()
  module <module> line 44
    self.peao = Peao(self)
  module <module> line 67
    self.peao = Elemento(PAWN, x=20, y=70, w=80, h= 80,
AttributeError: 'Peao' object has no attribute 'direita'
'''},
{'date': 'Tue Oct 10 2023 12:42:43.746 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Executado no modulo julia
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 82
    IlhaProibida()
  module <module> line 44
    self.peao = Peao(self)
  module <module> line 69
    self.terreno = terrenos[1]
NameError: name 'terrenos' is not defined
'''},
{'date': 'Tue Oct 10 2023 13:07:06.893 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 44
  px=1
  ^
IndentationError: unexpected indent
'''},
{'date': 'Tue Oct 10 2023 13:07:30.566 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 53
  self.terrenos = [Terreno(cena=oceano, posy=py, posx=px*110+10, local=info_terrenos[i])
                                                                                                                                                                                                                                                    ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Oct 10 2023 13:08:34.723 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 53
  self.terrenos = [Terreno(cena=oceano, posy=py, posx=px*110+10, local=self.info_terrenos[i])
                                                                                                                                                                                                                                                         ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Oct 10 2023 13:10:02.165 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Executado no modulo julia
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 95
    IlhaProibida()
  module <module> line 50
    if x==7:
NameError: name 'x' is not defined
'''},
{'date': 'Tue Oct 10 2023 13:12:24.487 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Executado no modulo julia
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 95
    IlhaProibida()
  module <module> line 50
    if x==7:
NameError: name 'x' is not defined
'''},
{'date': 'Tue Oct 10 2023 13:13:50.813 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Executado no modulo julia
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 96
    IlhaProibida()
  module <module> line 51
    if x==7:
NameError: name 'x' is not defined
'''},
{'date': 'Tue Oct 10 2023 13:17:17.821 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Executado no modulo julia
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 97
    IlhaProibida()
  module <module> line 51
    if x==7:
NameError: name 'x' is not defined
'''},
{'date': 'Tue Oct 24 2023 09:52:16.138 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Executado no modulo julia
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 97
    IlhaProibida()
  module <module> line 51
    if x==7:
NameError: name 'x' is not defined
'''},
{'date': 'Tue Oct 24 2023 09:58:38.534 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 23
    from julia.main import IlhaProibida as Ilha
  module julia.main line 23
    from julia.main import IlhaProibida as Ilha
ImportError: cannot import name 'IlhaProibida'

ImportError
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 268
    action()
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 23
    from julia.main import IlhaProibida as Ilha
  module julia.main line 23
    from julia.main import IlhaProibida as Ilha
'''},
{'date': 'Tue Oct 24 2023 09:59:28.144 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''
 module <string> line 52
  def monta_tabuleiro_oceano(self)
                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Oct 24 2023 09:59:40.476 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 23
    from julia.main import IlhaProibida as Ilha
ImportError: cannot import name 'IlhaProibida'
'''},
{'date': 'Tue Oct 24 2023 10:00:27.618 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 23
    from julia.main import IlhaProibida as Ilha
ImportError: cannot import name 'IlhaProibida'
'''},
{'date': 'Tue Oct 24 2023 10:01:01.662 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 23
    from julia.main import IlhaProibida as Ilha
  module julia.main line 23
    from julia.main import IlhaProibida as Ilha
ImportError: cannot import name 'IlhaProibida'

ImportError
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 268
    action()
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 23
    from julia.main import IlhaProibida as Ilha
  module julia.main line 23
    from julia.main import IlhaProibida as Ilha
'''},
{'date': 'Tue Oct 24 2023 10:06:10.648 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 24
    from julia import main as jmain
ImportError: cannot import name 'main'

AttributeError
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 268
    action()
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 24
    from julia import main as jmain
'''},
{'date': 'Tue Oct 24 2023 11:08:00.646 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 117
    IlhaProibida()
  module <module> line 46
    self.monta_tabuleiro_oceano()
  module <module> line 61
    self.terrenos = [Terreno(cena=oceano, posy=py, posx=px*110+10, local=lc)]
NameError: name 'lc' is not defined
'''},
{'date': 'Tue Oct 24 2023 11:09:58.967 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 23
    from anastasia import main as amain
ImportError: cannot import name 'main'

ImportError
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 268
    action()
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 23
    from anastasia import main as amain
  module anastasia.main line 35
    from _spy.circus.game import Circus
'''},
{'date': 'Tue Oct 24 2023 11:10:33.942 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 117
    IlhaProibida()
  module <module> line 46
    self.monta_tabuleiro_oceano()
  module <module> line 61
    self.terrenos = [Terreno(local=lc, posy=py, posx=px*110+10,cena=oceano)]
NameError: name 'lc' is not defined
'''},
{'date': 'Tue Oct 24 2023 11:12:34.910 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 117
    IlhaProibida()
  module <module> line 48
    self.terrenos[1].ocupa(self.peao)
IndexError: list index out of range
'''},
{'date': 'Tue Oct 24 2023 11:27:07.432 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 117
    IlhaProibida()
  module <module> line 48
    self.terrenos[1].ocupa(self.peao)
IndexError: list index out of range
'''},
{'date': 'Tue Oct 24 2023 11:43:09.609 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 117
    IlhaProibida()
  module <module> line 48
    self.terrenos[1].ocupa(self.peao)
IndexError: list index out of range
'''},
{'date': 'Tue Oct 24 2023 11:46:37.193 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 116
    IlhaProibida()
  module <module> line 48
    self.terrenos[1].ocupa(self.peao)
IndexError: list index out of range
'''},
{'date': 'Tue Oct 24 2023 11:51:27.401 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 115
    IlhaProibida()
  module <module> line 47
    self.peao = Peao(self)
  module <module> line 90
    self.terreno = self.terrenos.index(1)
AttributeError: 'Peao' object has no attribute 'terrenos'
'''},
{'date': 'Tue Oct 24 2023 11:52:53.798 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 115
    IlhaProibida()
  module <module> line 47
    self.peao = Peao(self)
  module <module> line 90
    self.terreno = self.terrenos.index[1]
AttributeError: 'Peao' object has no attribute 'terrenos'
'''},
{'date': 'Tue Oct 24 2023 11:54:23.155 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 115
    IlhaProibida()
  module <module> line 47
    self.peao = Peao(self)
  module <module> line 90
    self.terreno = self.terrenos.index[1]
AttributeError: 'Peao' object has no attribute 'terrenos'
'''},
{'date': 'Tue Oct 24 2023 11:55:15.983 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 115
    IlhaProibida()
  module <module> line 47
    self.peao = Peao(self)
  module <module> line 90
    self.terreno = self.terrenos.index[1]
AttributeError: 'Peao' object has no attribute 'terrenos'
'''},
{'date': 'Tue Oct 24 2023 11:56:25.1 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 116
    IlhaProibida()
  module <module> line 48
    self.terrenos[1].ocupa(self.peao)
IndexError: list index out of range
'''},